packages/twenty-front/src/modules/object-record/record-relation-card/components/RecordRelationFieldCardSection.tsx

```diff
@@ -263,13 +248,15 @@ export const RecordRelationFieldCardSection = () => {
         </StyledHeader>
         {!!relationRecords.length && (
           <Card>
-            {relationRecords.slice(0, 5).map((relationRecord, index) => (
-              <RecordRelationFieldCardContent
-                key={`${relationRecord.id}${relationLabelIdentifierFieldMetadata?.id}`}
-                divider={index < relationRecords.length - 1}
-                relationRecord={relationRecord}
-              />
-            ))}
+            {relationRecords
+              .slice(0, 5)
+              .map((relationRecord: any, index: number) => (
```

Comment:
With my suggestion above, we can remove the types!

```suggestion
              .map((relationRecord, index) => (
```